arrow-core-data/src/test/kotlin/arrow/core/EitherTest.kt

```diff
@@ -216,5 +216,16 @@ class EitherTest : UnitSpec() {
           Right(a).handleErrorWith { Right(b) } == Right(a)
       }
     }
+
+    "catch should return Valid(result) when f does not throw" {
```

Comment:
Thanks, we were missing these. I should have added them while working on `Validated.catch {}`, I think I noticed `Either` was missing these but totally forgot to add them in the end 🤦